zephyr/src/main.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdio.h>
+
+#include "vendor_specific.h"
+#include "eloop.h"
+#include "indigo_api.h"
+#include "utils.h"
+
+#include <zephyr/posix/pthread.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(wfa_qt, LOG_LEVEL_DBG);
```

Comment:
Define a separate logger and use

Reply:
Cant'we use this? Any specific reason for new logger?

Reply:
Yes, this is too generic and could enable other modules logs too. We need an option to enable logs only for this module.

Reply:
@krish2718 Added, please review.